tests/fixtures/virt/osinfo.py

```
@@ -0,0 +1,41 @@
+
+
+@pytest.fixture(scope="module")
+def osinfo_repo():
+    return f"{py_config['servers']['https_server']}/cnv-tests/osinfo-db/"
+
+
+@pytest.fixture(scope="module")
+def latest_osinfo_db_file_name(osinfo_repo):
+    sorted_osinfo_repo = f"{osinfo_repo}/?C=M;O=A"
+    soup_page = BeautifulSoup(
+        markup=requests.get(sorted_osinfo_repo, headers=get_artifactory_header(), verify=False).text,
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)